hadoop-ozone/dist/src/main/compose/compatibility/test.sh

```diff
@@ -20,7 +20,7 @@ export COMPOSE_DIR
 
 export SECURITY_ENABLED=false
 export OZONE_REPLICATION_FACTOR=1
-
+HDDS-6792
```

Comment:
I'm not sure whether it is intended or not, I didn't found other similar usages, or just a typo?

Reply:
It's a typo.